h5py/_hl/selections.py

```diff
@@ -91,6 +91,68 @@ def select(shape, args, dsid):
     return sel
 
 
+class MultiBlockSlice(object):
+
+    """
+        A conceptual extension of the built-in slice object to allow selections
+        using start, stride, count and block.
+
+        If given, these parameters will be passed directly to
+        H5Sselect_hyperslab. The defaults are start=0, stride=1, block=1,
+        count=length, which will select the full extent.
+
+        __init__(start, count, stride, block) => Create a new MultiBlockSlice, storing
+            any given selection parameters and using defaults for the others
+        start => The offset of the starting element of the specified hyperslab
+        count => The number of blocks to select
+        stride => The number of elements between the start of one block and the next
+        block => The number of elements in each block
+
+    """
+
+    def __init__(self, start=0, count=None, stride=1, block=1):
+        if start < 0:
+            raise ValueError("Start must be positive")
+        if count is not None and count < 1 or stride < 1 or block < 1:
```

Comment:
```suggestion
        if (count is not None and count < 1) or stride < 1 or block < 1:
```

It's probably correct for the computer anyway, but I basically always want parentheses where the precedence matters, so I don't have to think about how Python evaluates it.